gateway-service-baseline-lib/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+    `java-library`
+    jacoco
```

Comment:
Which plugin publishes this jar ? Its needed right ?

Reply:
yeah, @sarthak77 please add `id("org.hypertrace.publish-plugin")`